packages/composite-checkout/src/components/button.js

```diff
@@ -5,23 +5,33 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 
+/**
+ * Internal Classes
+ */
+import joinClasses from '../lib/join-classes';
+
 export default function Button( {
+	className,
 	buttonState,
 	buttonType,
-	onClick,
-	className,
-	fullWidth,
+	isBusy,
 	children,
 	...props
 } ) {
+	const classNames = joinClasses( [
+		'checkout-button',
+		...( buttonState ? [ 'is-status-' + buttonState ] : [] ),
```

Comment:
`is-status-*`, `is-type-*`, and `checkout-button` aren't specifically needed for the PR, but I figured they'd help for future theming of the buttons.